src/process/conditions/__tests__/conditions.test.ts

```diff
@@ -1,142 +1,648 @@
-import { expect } from 'chai';
-import { processSync } from '../../process'
-import { conditionProcessInfo } from '../index';
-import { ConditionsScope, ProcessContext } from 'types';
-import { get } from 'lodash';
+import { expect } from "chai";
```

Comment:
Please reformat the file to use single qoutes

Reply:
done